.github/workflows/test.yml

```diff
@@ -16,5 +16,11 @@ jobs:
       run: go mod download
     - name: Build
       run: go build -v ./...
-    - name: TF tests
+    - name: Unit tests
+      run: go test -v -short -cover -parallel 4 ./...
+      env:
+        METAL_AUTH_TOKEN: ${{ secrets.METAL_AUTH_TOKEN }}
```

Comment:
The unit tests step is intended for tests that don't need an auth token.  Tests that need an auth token should skip if the tests are running in short mode.

```suggestion
```